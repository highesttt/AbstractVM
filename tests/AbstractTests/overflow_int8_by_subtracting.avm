push int8(-128)
push int8(1)
sub
dump
exit
