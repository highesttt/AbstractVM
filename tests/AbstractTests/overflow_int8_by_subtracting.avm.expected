Return: 84
Error: [1;31mline 3:[1;33m Overflow or underflow on a value
